ast/edit/task_editor_test.go

```diff
@@ -0,0 +1,432 @@
+package edit_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/ast/asttest"
+	"github.com/influxdata/flux/ast/edit"
+)
+
+func TestGetOption(t *testing.T) {
```

Comment:
Could we add a test here that represents the actual structure of a `option task = { ... }` statement (with all required fields)? I'm interested to see a flow of `GetOption` and then `GetProperty` to extract meaningful data from that statement. This could even just go in test `ExampleTaskOptionsExtraction` function 😄